web-app/src/app/tabs/playback/playback.component.css

```diff
@@ -14,6 +14,14 @@ input[type="checkbox"] {
   accent-color: #c2185b;
 }
 
+input[type="datetime-local"] {
+  padding: 12px, 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  font-size: large;
+  text-align: center;
+}
+
```

Comment:
can we have this instead:

input[type="datetime-local"] {
  padding: 5px 8px;
  margin: 15px 5px;
  display: inline-block;
  border: 1px solid rgb(50, 50, 50);
  border-radius: 4px;
  box-sizing: border-box;
  text-align: center;
  font-size: large;
  background-color: rgb(50, 50, 50);
  color: white;
}

it will look like this: 
<img width="931" alt="image" src="https://user-images.githubusercontent.com/29678619/174669084-f8217bac-793b-4971-9d82-c2353a6280aa.png">